set-search-api-client/src/main/java/eu/europeana/set/search/service/impl/SearchApiClientImpl.java

```diff
@@ -66,18 +80,31 @@ public SearchApiResponse searchItems(String uri, String apiKey, String action)
      * @throws JSONException
      */
     protected List<String> jsonArrayToStringArray(JSONArray valueObject) throws JSONException {
+	return jsonArrayToStringArrayByField(valueObject, WebUserSetFields.ID);
+    }
+
+    /**
+     * Get list of values specified by field name from JSONArray
+     * 
+     * @param valueObject
+     * @return list of values
+     * @throws JSONException
+     */
+    protected List<String> jsonArrayToStringArrayByField(JSONArray valueObject, String fieldName) throws JSONException {
```

Comment:
please rename the method to indicate what it does, extracting the item descriptions

Reply:
Method renamed in extractItemDescriptions